### PR TITLE
NR-34627: Aerospike- new quickstart

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary

**Ticket URL**: https://issues.newrelic.com/browse/NR-34627

**Description**: This PR has the new quickstart integration named "Aerospike"

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### Screenshot 

screencapture-newrelic-instant-observability-preview-2023-05-03-23_06_31

### CODE REVIEW

File: dashboards/aerospike/aerospike.json [hunk 1]
```diff
@@ -0,0 +1,1037 @@
...
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT latest(aerospike_node_stats_cluster_integrity) as ''"
```
Comment (mdumpati) [inline dashboards/aerospike/aerospike.json:356]:
```suggestion
                "query": "FROM Metric SELECT IF(latest(aerospike_node_stats_cluster_integrity)=1, 'True', 'False') as ''"
```

File: dashboards/aerospike/aerospike.json [hunk 2]
```diff
@@ -0,0 +1,1037 @@
...
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT latest(aerospike_node_stats_cluster_is_member) as ''  "
```
Comment (mdumpati) [inline dashboards/aerospike/aerospike.json:393]:
```suggestion
                "query": "FROM Metric SELECT IF(latest(aerospike_node_stats_cluster_is_member)=1, 'True', 'False') as ''  "
```

File: dashboards/aerospike/aerospike.json [hunk 3]
```diff
@@ -0,0 +1,1037 @@
...
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT max(aerospike_node_stats_time_since_rebalance) as ''"
```
Comment (mdumpati) [inline dashboards/aerospike/aerospike.json:430]: Try to convert the result as a Hours, Minutes and Seconds.

File: dashboards/aerospike/aerospike.json [hunk 4]
```diff
@@ -0,0 +1,1037 @@
...
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT latest(aerospike_node_stats_threads_detached) as 'Detached', latest(aerospike_node_stats_threads_joinable) as 'joinable'"
```
Comment (mdumpati) [inline dashboards/aerospike/aerospike.json:101]:
```suggestion
                "query": "FROM Metric SELECT latest(aerospike_node_stats_threads_detached) as 'Detached', latest(aerospike_node_stats_threads_joinable) as 'Joinable'"
```

**Review by mdumpati — COMMENTED** [review]
@RamanaReddy8801 Please find my reviews or suggestions.

File: dashboards/aerospike/aerospike.json [hunk 5]
```diff
@@ -0,0 +1,1037 @@
...
+                "query": "FROM Metric SELECT latest(aerospike_node_stats_cluster_size) as '' "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Partitions",
```
[the remark this answers is not in the document]
Comment (RamanaReddy8801) [reply dashboards/aerospike/aerospike.json:153]: This chart shows two partitions like dead partitions and Unavailable partitions, not for Partitions Unavailable

File: dashboards/aerospike/aerospike.json (hunk 3 above)
Comment (RamanaReddy8801) [reply dashboards/aerospike/aerospike.json]: As per the manohar suggestion, results convert into Minutes

**Review by brammerl — APPROVED** [review]

**Comment by brammerl** [comment]
LG on my end. Going to wait for @mdumpati to also approve

**Comment by mdumpati** [comment]
Looks Good to me

**Review by mdumpati — APPROVED** [review]
Approved

**Comment by brammerl** [comment]
@RamanaReddy8801 seems like you need to sign the CLA then we can merge

**Pull request merged**